From 1a028a4935641ac4ffdcf807432721b6156c30bb Mon Sep 17 00:00:00 2001
From: Zheng Zengkai <zhengzengkai@huawei.com>
Date: Thu, 23 Dec 2021 21:31:53 +0800
Subject: [PATCH] openeuler_defconfig: Adjust CONFIG_NODES_SHIFT to 7
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 1a028a4935641ac4ffdcf807432721b6156c30bb


hulk inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4J6G3
CVE: NA

-------------------------------------------------

As phytium 4-processor server may have 32 NUMA nodes, in order
to support scalability in the future, adjust CONFIG_NODES_SHIFT
to 7 to increase maximum number of NUMA Nodes available on the
target system to 128 in advance for arm64 openeuler_defconfig.

Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Reviewed-by: Xie XiuQi <xiexiuqi@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 66124b991d96..3d66cdd5826a 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -384,7 +384,7 @@ CONFIG_NR_CPUS=4096
 CONFIG_HOTPLUG_CPU=y
 # CONFIG_ARM64_BOOTPARAM_HOTPLUG_CPU0 is not set
 CONFIG_NUMA=y
-CONFIG_NODES_SHIFT=4
+CONFIG_NODES_SHIFT=7
 CONFIG_USE_PERCPU_NUMA_NODE_ID=y
 CONFIG_HAVE_SETUP_PER_CPU_AREA=y
 CONFIG_NEED_PER_CPU_EMBED_FIRST_CHUNK=y
-- 
2.26.2

